### PR TITLE
EN-2224 (master) Managed null kie config for BpmCaseInstanceSelectorWidget

### PR DESCRIPTION
**Pull request by zonia3000**

### CODE REVIEW

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/apsadmin/portal/specialwidget/BpmBpmCaseInstanceSelectorWidgetAction.java [hunk 1]
```diff
@@ -141,10 +141,16 @@ protected String extractInitConfig() {
                     this.setFrontEndCaseData(frontEndCaseDatain);
                     JSONObject frontEndCaseDatainjs = new JSONObject(frontEndCaseDatain);
 
-                    this.setKnowledgeSourcePath(frontEndCaseDatainjs.getString("knowledge-source-id"));
+                    String configuredSourceId = frontEndCaseDatainjs.getString("knowledge-source-id");
```
Comment (joewhite101) [inline plugins/entando-plugin-jpkiebpm/src/main/java/org/entando/entando/plugins/jpkiebpm/apsadmin/portal/specialwidget/BpmBpmCaseInstanceSelectorWidgetAction.java]: Please add test coverage for the error condition

**Comment by entando-service** [comment]
All is Well

**Comment by entando-service** [comment]
All is Well

**Pull request merged**